modules/entity_meta_example/entity_meta_example.install

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @file
+ * Entity meta relation node module install file.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Implements hook_install().
+ */
+function entity_meta_example_install() {
+
+  // Loads all the bundles of node meta example relation.
```

Comment:
Guess comment is wrong. Also, not sure what this install hook does (as in why it's needed in this PR). Maybe help me understand?